addon/components/tags-widget/component.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+import layout from './template';
+
+export default Ember.Component.extend({
+    layout,
+    tags: [],
+    actions: {
+        addATag(tag) {
+            this.attrs.addATag(tag);
```

Comment:
Nice job on the DDAU approach. I think we'll want to use `sendAction` here (http://emberjs.com/api/classes/Ember.Component.html#method_sendAction), e.g.:

`this.sendAction('addATag', tag)`